plugins/mcreator-localization/help/fr_FR/gui/image_animated_frame.md

```diff
@@ -0,0 +1,3 @@
+Si la texture sélectionnée est tiled (plusieurs textures dans une bande verticale), cette procédure détermine

```

Comment:
```suggestion
Si la texture sélectionnée est carrelée (plusieurs textures dans une bande verticale), cette procédure détermine

```

Reply:
For Klemen's information: this will also be applied to the particle help tip since I picked this part of the French tip there.

Reply:
This should happen in PR called "fix particle help tip" as this PR is not about particles